Assets/_BForBoss/_Core/Scripts/Managers/BaseWorldManager.cs

```diff
@@ -68,6 +70,31 @@ protected virtual void OnValidate()
             }
         }
 
+#if UNITY_EDITOR
```

Comment:
The BaseWorldManager should only be handling game stuff, it shouldn't be handling anything related to Debugging.

This should be it's own script at the very least, even it's own prefab.

Reply:
Sure, can do. This will let me put the scripts in utility now too (Since I don't need to add a new reference to asmdef). 

But, in future, we should discuss this rule. cause in X months from now, if we wanted to add something like this and the two solutions were between adding `#If Unity_Editor` to a script or adding a prefab into 30 or 40+ scenes, I would personally think it is safer to just add the macro haha

Reply:
But if u have a prefab, you can just update that prefab?

Reply:
Basically, the question to ask is...What's the job of the BaseWorldManager. 

Since if we were to add `OpenLevelDesignFeedbackWindow`, then it's not just a worldManager anymore, it's also Debugging and editor tools.  And that breaks SOLID principles